plugins/repository-s3/src/test/java/org/elasticsearch/repositories/s3/AmazonS3Wrapper.java

```diff
@@ -143,12 +143,14 @@ public void setS3ClientOptions(S3ClientOptions clientOptions) {
     }
 
     @Override
-    public void changeObjectStorageClass(String bucketName, String key, StorageClass newStorageClass) throws AmazonClientException, AmazonServiceException {
```

Comment:
This file and its brother below are huge and we might be able to use mockito for it now but I didn't want to make a big change to how the test was implemented. And I was on an airplane and this is the kind of thing you do when you are tired and bored on an airplane. I'd be happy to be told "don't merge this, see about mockito instead" if anyone things that is a good idea.